### PR TITLE
fix(ui-library): text input-ensure consistency between storybook documentation and property names

### LINKED ISSUE #864: Text Input - Ensure Consistency between Storybook Documentation and Codebase Property Names
### Description / User story

As a developer on the team,
I want to ensure consistency between the property names defined in the Storybook documentation for Text Input component
So that the Text Input component codebase accurately reflects the documented property names.

This user story outlines the need to maintain consistency between the documentation and the codebase regarding component property names. Each component is checked to ensure the property names in the codebase match the corresponding names in the Storybook documentation. If the discrepancies are found, they are listed in[ this table](https://ts.accenture.com/:x:/r/sites/b01ler-core/_layouts/15/Doc.aspx?sourcedoc=%7Bb2cc9fb8-5c39-4049-9652-a4e6bbe09ef1%7D&action=editnew) on the page "TextInput". Column Current Code Property Name describes how is the property currently named in the code. Column Desired Code Property Name is how the property should be named. Properties marked green don't need adjustment.

### Acceptance Criteria

- [ ]  All Text Input component's property names that differ from Storybook documentation property names (listed in the above mentioned table) should be updated
- [ ] The updated property names should not introduce any unintended side effects or issues in the functionality of the components
- [ ] The library builds successfully, and there are no reported issues related to the property name updates 
- [ ] [Props Excel](https://ts.accenture.com/:x:/r/sites/b01ler-core/Shared%20Documents/Documentation_Props.xlsx?d=wb2cc9fb85c3940499652a4e6bbe09ef1&csf=1&web=1&e=bhRYao) has been updated / comments have been resolved and props changed from red to black font color > @thrbnhrtmnn or @angelicahoyss can support here

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by RubirajAccenture**
closes #864

### CODE REVIEW

File: packages/ui-library/src/components/input-field-text/index.stories.ts
Comment (davidken91) [inline packages/ui-library/src/components/input-field-text/index.stories.ts:114]: I think we can remove these comments from index.stories.ts

File: packages/ui-library/src/components/input-field-text/index.stories.ts
Comment (davidken91) [inline packages/ui-library/src/components/input-field-text/index.stories.ts:196]: Once this is removed, then it should be ok to approve and merge

**Review by thrbnhrtmnn — COMMENTED** [review]
LGTM, once davids comments are resolved we can approve and merge

File: packages/ui-library/src/components/input-field-text/index.stories.ts
Comment (angsherpa456) [inline packages/ui-library/src/components/input-field-text/index.stories.ts:114]: Please maintain the consistency with the new-line. Right now they are little bit mixed up.

Comment (davidken91) [reply packages/ui-library/src/components/input-field-text/index.stories.ts:114]: This is ready for approval / merging once this is resolved

**Review by angsherpa456 — DISMISSED** [review]
LGTM

**Review by thrbnhrtmnn — CHANGES_REQUESTED** [review]
Hey @RubirajAccenture , just one small fix needed:

Please change the order of the "name" and the "inputFieldTextId" props in the prop panel. "inputFieldTextId" should come first, then "name".

File: packages/ui-library/src/components/input-field-text/index.stories.ts
Comment (angsherpa456) [inline packages/ui-library/src/components/input-field-text/index.stories.ts:146]: Is this still relevant?

**Review by angsherpa456 — APPROVED** [review]
LGTM

**Pull request merged**
